File: Front/client/src/DTOs/PostDTO.ts
import { ImageDTO } from "./ImageDTO";
import { v4 as uuidv4 } from 'uuid';
export class PostDTO{
    id:number;
    guid: string;
    title:string;
    description:string;
    images: ImageDTO[];
    userId:number;
    constructor(){
        this.id = 0;
        this.title = "";
        this.description = "";
        this.userId = 0
        this.images = []
        this.guid = "";
    }
    generatedGuid(){
        this.guid = uuidv4()
    }
}

File: BackEnd/Api/DTOs/PostDTO.cs
using Api.DTOs;

namespace Api.ViewModel
{
    public class PostDTO
    {
        public int id { get; set; }
        public string guid {get; set;}
        public string title { get; set; }
        public string description { get; set; }
        public List<ImageDTO> images { get; set; }
        public int userId { get; set; }
    }
}
